terraform/root_common.tf

```diff
@@ -0,0 +1,307 @@
+  policy_attachments = { task_policy = module.jenkins_integration_task_policy.policy_arn, cloudwatch_policy = module.jenkins_integration_task_cloudwatch_policy.policy_arn }
+}
+
+module "jenkins_integration_task_policy" {
+  source        = "./tdr-terraform-modules/iam_policy"
+  name          = "TDRJenkinsTaskPolicy${title(local.environment)}"
+  policy_string = templatefile("./tdr-terraform-modules/iam_policy/templates/jenkins_ecs_task_integration.json.tpl", { account_id = data.aws_caller_identity.current.account_id })
+}
+
+module "jenkins_integration_task_cloudwatch_policy" {
+  source        = "./tdr-terraform-modules/iam_policy"
+  name          = "TDRJenkinsCloudwatchPolicyMgmt"
+  policy_string = templatefile("./tdr-terraform-modules/iam_policy/templates/jenkins_ecs_task_integration_cloudwatch.json.tpl", { account_id = data.aws_caller_identity.current.account_id })
+}
+
+module "jenkins_integration_execution_role" {
+  source             = "./tdr-terraform-modules/iam_role"
+  common_tags        = local.common_tags
+  name               = "TDRJenkinsAppExecutionRole${title(local.environment)}"
+  assume_role_policy = templatefile("./tdr-terraform-modules/ecs/templates/ecs_assume_role_policy.json.tpl", {})
+  policy_attachments = { execution_policy = module.jenkins_integration_execution_policy.policy_arn, cloudwatch_policy = module.jenkins_integration_task_cloudwatch_policy.policy_arn }
+}
+
+module "jenkins_integration_execution_policy" {
+  source        = "./tdr-terraform-modules/iam_policy"
+  name          = "TDRJenkinsExecutionPolicyMgmt"
+  policy_string = templatefile("./tdr-terraform-modules/iam_policy/templates/jenkins_ecs_execution_integration.json.tpl", { account_id = data.aws_caller_identity.current.account_id })
+}
+
+module "ecr_jenkins_build_npm_repository" {
+  source           = "./tdr-terraform-modules/ecr"
+  name             = "jenkins-build-npm"
+  image_source_url = "https://github.com/nationalarchives/tdr-jenkins/blob/master/docker/npm/Dockerfile"
+  common_tags      = local.common_tags
+  policy_name      = "jenkins_policy"
+  policy_variables = { role_arn = module.jenkins_build_npm_execution_role.role.arn }
+}
+
+module "jenkins_build_npm_execution_policy" {
+  source        = "./tdr-terraform-modules/iam_policy"
+  name          = "TDRJenkinsBuildNpmExecutionPolicy"
+  policy_string = templatefile("./tdr-terraform-modules/iam_policy/templates/jenkins_ecr_policy.json.tpl", { repository_arn = module.ecr_jenkins_build_npm_repository.repository.arn })
+}
+
+module "jenkins_build_npm_execution_role" {
+  source             = "./tdr-terraform-modules/iam_role"
+  assume_role_policy = templatefile("./tdr-terraform-modules/ecs/templates/ecs_assume_role_policy.json.tpl", {})
+  common_tags        = local.common_tags
+  name               = "TDRJenkinsBuildNpmExecutionRole"
+  policy_attachments = { ecr_policy = module.jenkins_build_npm_execution_policy.policy_arn }
+}
+
+module "ecr_jenkins_build_aws_repository" {
```

Comment:
Should this be renamed `jenkins_ecr_build_aws_repository` to make it consistent with the other module names?